Add and link glossary terms UID and intid (#1466)
plone/documentation: docs/glossary.md

Content Delivery Network
CDN
    A Content Delivery Network (CDN) is a network of servers located in various geographic regions that work together to deliver web content to users quickly and efficiently.

> UID

Reviewer: We can stack terms.

```suggestion
unique identifier
UID
```

>    UID is an abbreviation meaning "User Identifier." On a Unix-like operating system, it is a unique numerical identifier assigned to each user. The system uses the UID to identify the user and determine access permissions to files, directories, and other system resources.

Reviewer: In this context, we are assigning a unique identifier to an image object, which is independent of a user identifier.

```suggestion
   UID is an acronym meaning "unique identifier".
   A UID is an identifier that is guaranteed to be unique among all identifiers used for those objects and for a specific purpose.
```

> intid

Reviewer: Where's the linked term?

> intid

Reviewer: ```suggestion
integer identifier
intid
```

>     In Plone, a "intid" is an integer identifier that is used to uniquely identify content objects within a Plone site. Each content item in a Plone site is given a unique intid, which the system uses internally to monitor and manage content.

Reviewer: ```suggestion
    In Plone, an integer identifier, or intid, is used to uniquely identify content objects within a Plone site.
    Each content item in a Plone site is given a unique intid, which the system uses internally to monitor and manage content.
```

Reviewer 2: "monitor and manage content"? 
I would say: reference content, keep track of link-integrity, link translations and related.

Content Delivery Network
CDN
    A Content Delivery Network (CDN) is a network of servers located in various geographic regions that work together to deliver web content to users quickly and efficiently.

unique identifier
UID
   UID is an acronym meaning "unique identifier".
   A UID is an identifier that is guaranteed to be unique among all identifiers used for those objects and for a specific purpose.

integer identifier
intid
    In Plone, an integer identifier, or intid, is used to uniquely identify content objects within a Plone site.

>     Each content item in a Plone site is given a unique intid, which the system uses internally to monitor and manage content.

Reviewer: Per @jensens's comment.

```suggestion
    Each content item in a Plone site is given a unique intid, which the system uses internally to reference content, keep track of link-integrity, link translations, and other related purposes.
```
